utils/handler.ts

```diff
@@ -73,3 +73,42 @@ export const handleCopy = async (value: string) => {
 export const handleNftImageLoadError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
   e.currentTarget.src = '/images/nft-placeholder.svg'
 }
+
+// return a sorter array
+export const handleSorterArrayAboutPath = (url, sorters, sorterValueEnum = null) => {
+  const params = url.slice(url.indexOf('?') + 1)
+  const sorterParamsArray = []
+
+  const searchParams = new URLSearchParams(params)
+  Array(...searchParams.keys()).forEach((item, index) => {
```

Comment:
Instanttiate an array with `Array([...])` is not recommended(although I cannot find the reference now)

And `searchParams` is iterable, so `forEach` can be found on it.

Reply:
`map` is clearer than `forEach` in this case